src/figures/figureW.jl

```diff
@@ -126,18 +129,27 @@ function L0fSearchHeatmap(df, dataType, vmax, clmin, clmax; murine = true)
     return pl
 end
 
-function plotSynergy(L0, f; murine::Bool, fit = nothing, Cellidx = nothing, quantity = nothing, c1q = false, neutralization = false)
+function plotSynergy(L0, f; murine::Bool, fit = nothing, Cellidx = nothing, Recepidx = false, Rbound = false, quantity = nothing, c1q = false, neutralization = false)
     Kav_df = importKav(; murine = murine, IgG2bFucose = murine, c1q = c1q, retdf = true)
     Kav = Matrix{Float64}(Kav_df[!, murine ? murineFcgR : humanFcgR])
-    ActI = murine ? murineActI : humanActI
 
-    if Cellidx == nothing #Not using single cell
-        FcExpr = importRtot(; murine = murine)
-    else #Using single cell
+    if Recepidx != nothing # look at only one receptor
+        FcExpr = zeros(length(Receps))
+        FcExpr[Recepidx] = importRtot(murine = murine)[Recepidx, Cellidx]
+        ylabel = "Activity"
+    elseif Cellidx != nothing # look at only one cell FcExpr
         FcExpr = importRtot(murine = murine)[:, Cellidx]
+        ylabel = "Activity"
+    else
+        FcExpr = importRtot(; murine = murine)
+    end
+    if fit == nothing
+        title = "Not fit"
+    else
+        title = "dataType"
```

Comment:
@stephendl102 I'm not sure what you wanted to do here. You want the word "not fit" and "dataType" in the parenthesis on the plot?

Reply:
@cyrillustan As it stands currently, synergy can be calculated from a mixture with a guess of activities vector ([-1,0,1,1]), or a disease model is provided and the activities are fit during regression. I wanted to be able to distinguish this on the plot itself so as not to confuse when regressed activities are used or not.